addons/binding/org.openhab.binding.hdanywhere/OSGI-INF/HDanywhereHandlerFactory.xml

```diff
@@ -10,11 +10,13 @@
 
 -->
 <scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="true" name="org.openhab.binding.hdanywhere.internal.HDanywhereHandlerFactory">
-
+  
    <implementation class="org.openhab.binding.hdanywhere.internal.HDanywhereHandlerFactory"/>
-
+   <reference bind="setItemChannelLinkRegistry" cardinality="1..1" interface="org.eclipse.smarthome.core.thing.link.ItemChannelLinkRegistry" name="ItemChannelLinkRegistry" policy="static" unbind="unsetItemChannelLinkRegistry"/>
```

Comment:
I can't say that I know enough to have a fully informed opinion, but with that huge caveat, no I don't know of a reason! ;-)